### PR TITLE
Possible workaround for #796

### PR DESCRIPTION
**Pull request by esurharun**
### Background

Please check out issue #796 

### Changes

Just applied OpenAI cookbook at https://github.com/openai/openai-cookbook/blob/main/examples/Embedding_long_inputs.ipynb

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have thoroughly tested my changes with multiple different prompts.
- [ ] I have considered potential risks and mitigations for my changes.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have not snuck in any "extra" small tweaks changes 

I just coded to speed up to the full fix. It needs to get revized and considered for Azure option too.

### CODE REVIEW

**Review by Sean19899999 — APPROVED** [review]

**Review by Sean19899999 — COMMENTED** [review]
hope it's useful

File: scripts/memory/base.py [hunk 1]
```diff
@@ -2,16 +2,61 @@
...
+
+
 
 def get_ada_embedding(text):
     text = text.replace("\n", " ")
     if cfg.use_azure:
         return openai.Embedding.create(input=[text], engine=cfg.get_azure_deployment_id_for_model("text-embedding-ada-002"))["data"][0]["embedding"]
     else:
-        return openai.Embedding.create(input=[text], model="text-embedding-ada-002")["data"][0]["embedding"]
+        if (len(text) > EMBEDDING_CTX_LENGTH):
```
Comment (Berserker66) [inline scripts/memory/base.py]:
```suggestion
        if len(text) > EMBEDDING_CTX_LENGTH:
```

**Comment by nponeccop** [comment]
@esurharun There are conflicts now

**Comment by nponeccop** [comment]
@esurharun Sorry it's our bot went crazy. We'll sort it out.

**Comment by Pwuts** [comment]
Fixed in #2542. Thanks for submitting though!

**Pull request closed without merging**